pallets/rc-migrator/src/lib.rs

```diff
@@ -1055,7 +1082,7 @@ pub mod pallet {
 		#[pallet::call_index(4)]
 		#[pallet::weight(T::RcWeightInfo::resend_xcm())]
 		pub fn resend_xcm(origin: OriginFor<T>, query_id: u64) -> DispatchResultWithPostInfo {
-			Self::ensure_admin_or_manager(origin)?;
+			ensure_root(origin)?;
```

Comment:
a duplication from the https://github.com/polkadot-fellows/runtimes/pull/929 PR, but required to have the right roles docs.